GVFS/GVFS.UnitTests/Git/GitAuthenticationTests.cs

```diff
@@ -262,19 +262,19 @@ private MockGitProcess GetGitProcess()
             int approvals = 0;
             int rejections = 0;
             gitProcess.SetExpectedCommandResult(
-                "-c credential.useHttpPath=true credential fill",
+                "-c credential.\"https://dev.azure.com\".useHttpPath=true credential fill",

```

Comment:
Add `const` for `-c credential.\"https://dev.azure.com\".useHttpPath=true` to use in all the tests?